### PR TITLE
Visualise core access

### PR DESCRIPTION
**Pull request by dougajmcdonald**
This PR add's a grid which shows the core access in R/W text in a grid as steps are taken.

Not super exciting visually, but means to an end!

image

### CODE REVIEW

File: src/tests/simulator/cell.test.js [hunk 1]
```diff
@@ -0,0 +1,40 @@
+
+import React from 'react';
+import { shallow, mount } from 'enzyme';
+import { expect } from 'chai';
+
+import Cell from '../../containers/simulator/cell';
+
+it('renders without crashing', () => {
```
Comment (gareththegeek) [inline src/tests/simulator/cell.test.js]: Do you not need a `describe` call before the `it`? e.g.

```
describe("Filter", () => {
    it("Does not modify tokens if no empty lines or END found", () => {
```

Comment (dougajmcdonald) [reply src/tests/simulator/cell.test.js]: Nope, it's optional kind of like a namespace.
The base test setup is from the create-react-app (CRA) template but I've brought in enzyme from Airbnb which is awesome

Comment (gareththegeek) [reply src/tests/simulator/cell.test.js]: I thought the `describe` call added the name of the test suite to each test so in my example, the test runner would output `Filter Does not modify tokens if....`.  Is it easy to tell which test file is failing without it?

**Comment by dougajmcdonald** [comment]
Yeah you're right, but it's still optional

**Pull request merged**